app/forms/steps/case/hearing_details_form.rb

```diff
@@ -0,0 +1,23 @@
+module Steps
+  module Case
+    class HearingDetailsForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+      has_one_association :case
+
+      attribute :hearing_court_name, :string
+      attribute :hearing_date, :multiparam_date
+
+      validates :hearing_court_name, presence: true
+      validates :hearing_date, presence: true,
+                multiparam_date: { allow_past: false, allow_future: true }
```

Comment:
Wondering if there is any limit on the future? I guess more than a few years in the future would be too many? 🤔

Reply:
For now I think let's go with the defaults (2050 is the default). Don't think we will get a definite answer. The validation is mostly for fat finger typos.

Reply:
This is currently set to 2050 here: https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/ed9c766a3fbe9c35eb1ad10738fb8a42b6016d6d/app/validators/multiparam_date_validator.rb#L4-L9

Perhaps we could also consider using age rather than fixed dates for this validation. I.e. 
```suggestion
             latest_years_from_now: 120
```

Reply:
Agree with @timpeat here - something we should note down to improve in a future iteration